Assets/Feature01/Scripts/Health/HealthManager.cs

```diff
@@ -0,0 +1,40 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class HealthManager : MonoBehaviour
+{
+
+
```

Comment:
I think you should send in the stats inside here. Inject it. Because it will change as your states change, then use the add remove health from this class. So finally, the stats will always represent the data you need to see